### DISCUSSION TITLE
Safari Extesion for iOS

### DISCUSSION THREAD

**Discussion by yzmoto** [simple-login/app#922] [Feature Requests]
I would like to see a safari extension for iOS. When you select an email address field, the extension would auto fill a new unique alias or let you create a custom alias instead of going to the app, creating an alias, copying it to your clipboard, going back to site and pasting it in.  Thank you.

**Comment by nguyenkims** [comment]
We are currently working on the new version of our iOS app that'd make creating a new alias easier, please stay tuned when this version is out!
